<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WorldParameters</class>
 <widget class="QDialog" name="WorldParameters">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>628</width>
    <height>505</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Parameters</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>618</width>
     <height>486</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <item row="10" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Meat energy (1-300)</string>
            </property>
           </widget>
          </item>
          <item row="9" column="2">
           <widget class="QLineEdit" name="Plants_energy">
            <property name="text">
             <string>50</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>Max age (1-10000)</string>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Number of gens of action (1-46)</string>
            </property>
           </widget>
          </item>
          <item row="5" column="2">
           <widget class="QLineEdit" name="Max_age">
            <property name="text">
             <string>1000</string>
            </property>
           </widget>
          </item>
          <item row="7" column="2">
           <widget class="QLineEdit" name="Min_energy_for_division">
            <property name="text">
             <string>60</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Initial live cell energy (1-1000)</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QLineEdit" name="Initial_live_cell_energy">
            <property name="text">
             <string>50</string>
            </property>
           </widget>
          </item>
          <item row="6" column="2">
           <widget class="QLineEdit" name="Number_of_gens_of_action">
            <property name="text">
             <string>10</string>
            </property>
           </widget>
          </item>
          <item row="10" column="2">
           <widget class="QLineEdit" name="Meat_energy">
            <property name="text">
             <string>50</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QLineEdit" name="Number_of_live_cells">
            <property name="text">
             <string>100</string>
            </property>
           </widget>
          </item>
          <item row="14" column="0">
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>Tree propagation speed</string>
            </property>
           </widget>
          </item>
          <item row="8" column="2">
           <widget class="QLineEdit" name="Number_of_plants">
            <property name="text">
             <string>1000</string>
            </property>
           </widget>
          </item>
          <item row="12" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Speed</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Number of live cells (1-5000)</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_14">
            <property name="text">
             <string>Name</string>
            </property>
           </widget>
          </item>
          <item row="7" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Min energy for division (1-600)</string>
            </property>
           </widget>
          </item>
          <item row="13" column="2">
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>1                                                                            3</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>Max live cell energy (1-1000)</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLineEdit" name="Name">
            <property name="text">
             <string>New world</string>
            </property>
           </widget>
          </item>
          <item row="14" column="2">
           <widget class="QSlider" name="Tree_propagation_speed">
            <property name="mouseTracking">
             <bool>true</bool>
            </property>
            <property name="tabletTracking">
             <bool>false</bool>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>3</number>
            </property>
            <property name="pageStep">
             <number>1</number>
            </property>
            <property name="value">
             <number>2</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="8" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Number of plants (1-9000)</string>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QLineEdit" name="Max_live_cell_energy">
            <property name="text">
             <string>200</string>
            </property>
           </widget>
          </item>
          <item row="9" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Plants energy (1-200)</string>
            </property>
           </widget>
          </item>
          <item row="11" column="2">
           <widget class="QLabel" name="label_13">
            <property name="text">
             <string>1                                                                           10</string>
            </property>
           </widget>
          </item>
          <item row="12" column="2">
           <widget class="QSlider" name="Speed">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>10</number>
            </property>
            <property name="pageStep">
             <number>1</number>
            </property>
            <property name="value">
             <number>5</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="Start">
          <property name="text">
           <string>Start</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
